### PR TITLE
VM queries added

### PR DESCRIPTION
**Pull request by ReneHezser**
# Overview/Summary

Added 2 Resource Graph Queries for VMs.

## This PR fixes/adds/changes/removes

none

### Breaking Changes

none

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/networking/load-balancer/code/lb-2/lb-2.kql
Comment (rodrigosantosms) [inline docs/content/services/networking/load-balancer/code/lb-2/lb-2.kql:1]: With Load Balancer I believe we have to two check two things, No BackEndPool, and BackEndPool IPs <=1. This query helps capturing both:

Resources
| where type =~ 'Microsoft.Network/loadBalancers'
| extend bep = properties.backendAddressPools
| extend BackEndPools = array_length(bep)
| where BackEndPools == 0
| project recommendationId = "lb-2", name, id, BackEndPools, BackendAddresses=0
| union (Resources
        | where type =~ 'Microsoft.Network/loadBalancers'
        | extend bep = properties.backendAddressPools
        | extend BackEndPools = array_length(bep)
        | mv-expand bip = properties.backendAddressPools
        | extend BackendAddresses = array_length(bip.properties.loadBalancerBackendAddresses)
        | where BackendAddresses <= 1
        | project recommendationId = "lb-2", name, id, BackEndPools, BackendAddresses)

File: docs/content/services/storage/storage-Account/_index.md
Comment (rodrigosantosms) [inline docs/content/services/storage/storage-Account/_index.md:1]: This recommendation was merged with another one,  @Dost2010 did you submit your PR with the merge?

File: docs/content/services/compute/virtual-machines/code/vm-19/vm-19.kql
Comment (rodrigosantosms) [inline docs/content/services/compute/virtual-machines/code/vm-19/vm-19.kql:1]: Please always project in this sequence:
RecommendationId="ABC14", AffectedResourceName, id, and then any custom values.

**Review by rodrigosantosms — CHANGES_REQUESTED** [review]
Please see comments for each file.

**Comment by ReneHezser** [comment]
> Please see comments for each file.

Thanks for the comments. I've updated queries accordingly.

**Review by lachaves — APPROVED** [review]
ran hugo server, reviewed the content. Everything looks good.

**Review by lachaves — APPROVED** [review]
LGTM

**Review by rodrigosantosms — APPROVED** [review]

**Pull request merged**